pkg/proxy/ipvs/README.md

```diff
@@ -45,6 +77,18 @@ kubeProxy:
     mode: ipvs
 ...
 ```
+before running
+
+`kube init --config <path_to_kubeadm_config_file>`
+
+If you are using Kubernetes v1.8, you can also add the flag `--feature-gates=SupportIPVSProxyMode=true` in `kubeadm init` command
```

Comment:
@m1093782566 You mean v1.8 --> v1.10 in this line too? For Kubernetes v1.9 and later, the flag `--feature-gates=SupportIPVSProxyMode=true` has been removed in `kubeadm init` .

Reply:
Okay, could you please mentioned it in somewhere?

Reply:
:ok: